tests/interfaces/test_qnode_torch.py

```diff
@@ -500,6 +527,11 @@ def circuit(x, y):
                 ],
             ]
         )
+
+        if diff_method == "backprop":
+            # TODO: check why this differs from other interfaces

```

Comment:
Created https://github.com/PennyLaneAI/pennylane/issues/1607 to contain this issue.

Reply:
Thanks Antal!